Utils/Dataflow/README

```diff
@@ -71,14 +73,14 @@
 2. Tasks and data samples (DS) metadata integration (ProdSys2)
 ---
 
-009 ---> *                                | Get task metadata from ProdSys2 DB
-                                          |
-    * --> 091(in) --> 025 --> 016         | Append task metadata with that from
-     \                           \        |   Rucio (input DS) and Chicago ES
-      \                           \       |
-       \-> 091(out) -> 093 -> 095 --> **  | Get DS metadata from Rucio and AMI
-                                          |
-    ** ----> 019 -> 069                   | Prepare and upload metadata to ES
+009 ---> *                                  | Get task metadata from ProdSys2 DB
+                                            |
+    * -> 091(in) -> 025 -> 016 -> 017       | Append task metadata with that from
```

Comment:
This line got longer than 80 characters... nothing critical, but still breaks the style )
I meant (https://github.com/PanDAWMS/dkb/pull/255#discussion_r292450195) removing of leading spaces, something like this:
```
009 ---> *                                | Get task metadata from ProdSys2 DB
                                          |
  * -> 091(in) -> 025 -> 016 -> 017       | Append task metadata with that from
   \                             \        |   Rucio (input DS) and Chicago ES
    \                             \       |
     \-> 091(out) -> 093 -> 095 -----> ** | Get DS metadata from Rucio and AMI
                                          |
  ** ----> 019 -> 069                     | Prepare and upload metadata to ES
```

Reply:
Done. Side note - there are other lines longer than 80 characters, so the README should be revised (somewhere outside of this PR).

Reply:
Thank you. It tells me that we should do something like [this](https://trello.com/c/Q6U7I1IF) to avoid such a technical, in fact, discussions in PRs... sorry for not doing it earlier, just didn\`t really *think* about it.